pandas/core/indexes/base.py

```diff
@@ -1005,6 +1005,29 @@ def to_series(self, **kwargs):
                       index=self._shallow_copy(),
                       name=self.name)
 
+    def to_frame(self, index=True):
+        """
+        Create a DataFrame with the columns the levels of the Index
+
+        .. versionadded:: 0.21.0
+
+        Parameters
+        ----------
+        index : boolean, default True
+            return the Index as the index
```

Comment:
This was a little confusing to me on first read.  Maybe something along the lines of "use this Index as the index of the resulting DataFrame" would be a little more clear?

Reply:
Yeah...bad English copied and pasted.  Fixed.